app/src/main/java/com/kickstarter/ui/views/CommentCard.kt

```diff
@@ -129,20 +133,29 @@ class CommentCard @JvmOverloads constructor(
         binding.commentBody.setTextColor(ContextCompat.getColor(context, commentBodyTextColor))
     }
 
+    private fun shouldShowReplyButton(cardCommentStatus: CommentCardStatus) =
+        cardCommentStatus == CommentCardStatus.COMMENT_WITH_REPLIES ||
+            cardCommentStatus == CommentCardStatus.COMMENT_WITHOUT_REPLIES ||
+            cardCommentStatus == CommentCardStatus.FAILED_TO_SEND_COMMENT
+
     /*
      * To display replies count
      * binding.replies.text = String.format("%s (%d)",resources.getString(R.string.view_replies), replies)
      */
     fun setCommentReplies(replies: Int) {
-        binding.replies.isVisible = replies > 0
+        setViewRepliesVisibility(replies > 0)
+    }
+
+    fun setViewRepliesVisibility(isViewRepliesVisible: Boolean) {
+        binding.replies.isVisible = isViewRepliesVisible && isCommentEnabledThreads
```

Comment:
@hadia @leighdouglas @sunday-okpoluaefe if you need to update visibility of the View replies button of the reply button please call `setReplyButtonVisibility` do not set directly `binding.replies.isVisible` values